base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/editing/blocks/BlockDetailsViewModel.java

```diff
@@ -123,6 +123,10 @@ public void updateBlock() {
     	editingBlock.setIsVisible(visible);
     }
     
+    public void validate() {
```

Comment:
I don't think so, we need to call it after the buttons have been created for the dialog box. The viewModel is created before this. This is also why thecall to validate happens  inside the createButtonsForButtonBar method, which I agree is messy but I see no other easy way to hook into the end of the dialog being created